client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-pattern-pages.ts

```diff
@@ -82,26 +82,30 @@ const usePatternPages = (
 		pageSlugs = INITIAL_PAGES;
 	}
 
+	const getFeaturedPageOrFirstInCategory = ( name: string ) =>
```

Comment:
I was about to move this function to the utils file but here makes more sense to me.